clients/go/cmd/zbctl/main_test.go

```diff
@@ -101,7 +101,7 @@ var tests = []testCase{
 	},
 	{
 		name:       "deploy processes",
-		cmd:        strings.Fields("--insecure deploy testdata/model.bpmn testdata/job_model.bpmn --resourceNames=model.bpmn,job.bpmn"),
+		cmd:        strings.Fields("--insecure deploy testdata/model.bpmn testdata/job_model.bpmn testdata/deploy_form.form --resourceNames=model.bpmn,job.bpmn,deploy_form.form"),
```

Comment:
❌ This test case verifies usage of the deprecated `deploy` command. It only supports deploying bpmn processes. Forms can only be deployed using the `deploy resources` command. You'll need to move the addition of the form to the test case below.